lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/PipelineBroadcastReduceDriverAndTasks/PipelinedBroadcastReduceDriver.cs

```diff
@@ -19,20 +19,16 @@
 using System.Collections.Generic;
```

Comment:
GC code change should be applicable to any clients. Yes, we can leave this code as it is. But use the modified code is good for testing. 
Contextmanager will need to be modified, I agree, very possibly when we move to details. We will also need to create Evaluator and Task manager as described in the design document. If those can be generic, why make them specific? It would be much easier for us to test those in smaller cases such as broadcast and reduce.